modules/optics/arrow-optics/src/main/kotlin/arrow/optics/docs_wip.kt

```diff
@@ -0,0 +1,76 @@
+package arrow.optics
```

Comment:
why is this docs helper file (or that's how it looks like) inside of the optics module and not the docs one?